content/doc/developer/plugin-governance/deprecating-or-removing-plugin.adoc

```diff
@@ -28,7 +28,7 @@ This page describes the process.
 . Submit a pull request to the Update Center's https://github.com/jenkins-infra/update-center2/blob/master/resources/artifact-ignores.properties[artifact-ignores.properties] file
 . Archive the plugin's repository
 ** If you have admin permissions in the repository, it is possible to do it from the GitHub's web interface
-** Otherwise, create a link:https://issues.jenkins.io/[Jenkins Jira] ticket to archive the plugin's repository (project: `INFRA`, component: `github`)
+** Otherwise, create a link:https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=github-permissions&template=2-github-permissions.ymlhttps://issues.jenkins.io/[help desk] ticket to archive the plugin's repository
```

Comment:
```suggestion
** Otherwise, create a link:https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=github-permissions&template=2-github-permissions.yml[help desk] ticket to archive the plugin's repository
```